src/main/java/Event.java

```diff
@@ -0,0 +1,27 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+
+    public Event(String description, LocalDateTime from, LocalDateTime to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+
+    @Override
+    public String toFile() {
+        DateTimeFormatter outputFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+        return "E" + super.toFile() + " | " + from.format(outputFormat) + " - " + to.format(outputFormat);
+    }
+
+    @Override
+    public String toString() {
+        DateTimeFormatter outputFormat = DateTimeFormatter.ofPattern("MMM dd yyyy HHmm");
+        return "[E]" + super.toString() + " (from: " + this.from.format(outputFormat) + " to: " + this.to.format(outputFormat) + ")";
```

Comment:
This line is exceeding the character limit of 100 characters. You can break it into multiple lines like this.
```suggestion
        return "[E]" + super.toString() + " (from: " + this.from.format(outputFormat) 
                + " to: " + this.to.format(outputFormat) + ")";
```